types/express-formidable/express-formidable-tests.ts

```diff
@@ -1,9 +1,14 @@
-import express = require("express");
-import expform = require("express-formidable");
+import * as express from "express"
+import * as expform from "express-formidable";
```

Comment:
Hey @evanshortiss, can you revert these, and also remove the lint suppressions from this package's `tslint.json`?

Reply:
Think it's good now 👍 Out of curiosity, I thought `require` was not recommended, is that not the case?